lib/src/rules/use_super_parameters.dart

```diff
@@ -172,6 +172,7 @@ class _Visitor extends SimpleAstVisitor {
       bool match = false;
       for (var i = 0; i < constructorParams.length && !match; ++i) {
         var constructorParam = constructorParams[i];
+        if (constructorParam is FieldFormalParameter) continue;
```

Comment:
We should also skip `SuperFormalParameter`. We won't run into any yet, but don't want false positives after the feature has started being adopted.

Reply:
Good catch.